src/IO/S3/copyS3File.cpp

```diff
@@ -524,6 +524,14 @@ namespace
                     continue; /// will retry
                 }
 
+                if ((outcome.GetError().GetErrorType() == Aws::S3::S3Errors::REQUEST_TIMEOUT) && (retries < max_retries))
```

Comment:
Sometimes AWS S3 server respond on `PutObject` with the following error
```
AWSClient: AWSErrorMarshaller: Encountered AWSError 'RequestTimeout': Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.
```

Reply:
I'm not sure how upload will behave if it notices that it was already uploaded but we will get a new error in worst case.